.github/workflows/main.yml

```diff
@@ -113,30 +109,56 @@ jobs:
         targetPlatform:
           - StandaloneOSX
     steps:
-      - name: Checkout Repository
-        uses: actions/checkout@v3
+      - uses: actions/checkout@v3
         with:
           lfs: true
       - uses: actions/cache@v3
         with:
-          path: ${{ matrix.projectPath }}/Library
-          key: Library-${{ matrix.projectPath }}-${{ matrix.targetPlatform }}
-          restore-keys: |
-            Library-${{ matrix.projectPath }}-
-            Library-
+          path: Library
+          key: Library-${{ matrix.targetPlatform }}
       - uses: game-ci/unity-builder@v2
+        env:
+          UNITY_LICENSE: ${{ secrets.UNITY_LICENSE }}
+          UNITY_EMAIL: ${{ secrets.UNITY_EMAIL }}
+          UNITY_PASSWORD: ${{ secrets.UNITY_PASSWORD }}
         with:
-          projectPath: ${{ env.PROJECT_PATH }}
           targetPlatform: ${{ matrix.targetPlatform }}
-      - name: Upload Build
-        uses: actions/upload-artifact@v3
+      - uses: actions/upload-artifact@v3
         with:
           name: Build-${{ matrix.targetPlatform }}
           path: build
 
   buildWithWindows:
-    needs: [build, buildWithMac]
-    if: needs.checklicense.outputs.is_unity_license_set == 'true'
+    needs: [build]
+    name: Build for ${{ matrix.targetPlatform }}
+    runs-on: windows-2019
+    strategy:
+      fail-fast: false
+      matrix:
+        targetPlatform:
+          - WSAPlayer
```

Comment:
Why is WSA player getting a separate workflow from the rest?

Reply:
because of the run-on? I'm not sure but I think that's the reason.

Reply:
I remember running into the license limitations with my paid license, when I would try running StandaloneWindows, StandaloneWindows64, and WSAPlayer in parallel.
I split off WSAPlayer from the other 2 in my personal workflow to avoid that issue, but maybe that's not necessary for a free license?